src/skill-tree.toml

```diff
@@ -49,12 +49,6 @@ label = "[T]::eq"
 requires = ["for", "fuzzy-ptr-comparisons", "trait_impl"]
 items = []
 
-[[group]]
-name = "transmute"
-label = "transmute"
-requires = ["unconst_rules"]
-items = []
```

Comment:
We should probably add `const_fn_transmute`, or rather make it one node with `const_fn_union`.